### DISCUSSION TITLE
How do I bootstrap the include Option

### DISCUSSION THREAD

**Discussion by wiswesser** [fluxcd/flux2#2410] [Q&A] [answered]
The GitRepository CR supports including a different Repository, which is a great function we want to use. But anytime we would e.g. install a new flux version using "flux bootstrap", this include setting is removed again. This makes sense, since this setting is not specified in our command. Am I missing an option to add this include using bootstrap? Or am I doing something else wrong?

Kind Regards,
Caspar Wiswesser

**Comment by stefanprodan** [comment, marked as answer]
Add a new GitRepository to `clusters/my-cluster` that uses the same `secretRef` as the one made by bootstrap. Include all repos in this dedicated GitRepository. I would not recommend altering the root GitRepository in any way, but if you want to do that, then you can use a patch https://fluxcd.io/docs/installation/#customize-flux-manifests

**Reply by wiswesser** [reply]
Thanks for the quick reply!
So I create a new GitRepository, which is the same as the one created by bootstrap, except for the added include part? But then I also need a new Kustomization that points to the new GitRepository, and the GitRepository/Kustomization created by bootstrap would just do nothing? Or am I misunderstading something?

**Reply by stefanprodan** [reply]
The `flux-system` Kustomization (the one created by bootstrap) applies the manifests from `clusters/my-cluster` and the Flux definitions. Inside `clusters/my-cluster` you place Flux GitRepos, HelmRepos, Buckets and Kustomizations for your apps. Please see here an example: https://github.com/fluxcd/flux2-kustomize-helm-example/tree/main/clusters/production

**Reply by wiswesser** [reply]
Ahh I see where I got confused. I had changed the path (Using the --path Option of bootstrap) so the Original Kustomization was responsible for the entire cluster. Having different Kustomizations makes a lot of sense. Thanks for your help.
